languages/python/oso/polar/host.py

```diff
@@ -280,28 +280,15 @@ def to_polar(self, v):
                 }
         else:
             instance_id = None
-            repr_str = None
             import inspect
 
             if inspect.isclass(v):
                 if v in self.types:
                     instance_id = self.types[v].id
-                # BEGIN HACK:
-                # The polar core uses the .repr property to determine whether or not
-                # to allow Roles.role_allows to be called with unbound variables as
-                # arguments (only for sqlalchemy_oso)
-                # Because of this, we need to continue to send the repr for
-                # sqlalchemy_oso.roles.OsoRoles.Roles ONLY
-                if (
-                    "OsoRoles" in v.__qualname__
-                    and v.__module__ == "sqlalchemy_oso.roles"
-                ):
-                    repr_str = repr(v)
-                # END HACK
             val = {
                 "ExternalInstance": {
                     "instance_id": self.cache_instance(v, instance_id),
-                    "repr": repr_str,
+                    "repr": None,
```

Comment:
I think we can actually kill `repr` now--we were only keeping it around for roles. Can you confirm @gkaemmer?

Reply:
But not high priority, let's just add this as a task for next cycle.

Reply:
I added this as a config follow-up, self-resolving.